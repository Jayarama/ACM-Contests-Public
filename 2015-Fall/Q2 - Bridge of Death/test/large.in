Lancelot, Robin, Galahad, Arthur, Bedevere
What is your name?
What is your quest?
What is your favorite colour?
What is your name?
What is your quest?
What is the capital of Assyria?
What is your name?
What is your quest?
What is your favorite colour?
What is your name?
What is your quest?
What is the airspeed velocity of an unladen swallow?
What is your name?
What is your quest?
What is your favorite colour?
Robin, Galahad, Arthur
What is the airspeed velocity of an unladen swallow?
What is your name?
What is your quest?
What is the airspeed velocity of an unladen swallow?
What is your name?
What is your quest?
What is your name?
What is the airspeed velocity of an unladen swallow?
What is your quest?
