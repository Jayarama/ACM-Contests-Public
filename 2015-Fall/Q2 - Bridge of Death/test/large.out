Ser Lancelot of Camelot
I seek the grail
Blue
Ser Robin of Camelot
I seek the grail
I don't know that!
AAAHHHHHHHHH
Ser Galahad of Camelot
I seek the grail
Blue
AAAHHHHHHHHH
Arthur, King of the Britons
I seek the grail
African or European?
I don't know that!
AAAHHHHHHHHH
I don't know that!
AAAHHHHHHHHH
African or European?
